tests/containers/podman_netavark.pm

```diff
@@ -37,7 +37,7 @@ sub is_container_running {
 
 # clean up routine only for systems that run CNI as default network backend
 sub _cleanup {
-    return if is_alp;
+    return if is_netavark_default;
```

Comment:
IMO this should still cleanup, but revert to the system default state.

Reply:
I'm considering using snapshots for test runs where we change the default configuration instead of trying to reset it to what we think the state was before.

Reply:
If that works on all relevant platforms, sure.

Alternatively, doing `podman system reset -f` with moved away `storage.conf` should do it. Or `rm -r /var/lib/containers/*` if needed.

Reply:
This is not related here any more. I'll keep it in mind.